src/main/java/io/pivotal/cla/data/repository/IndividualSignatureRepository.java

```diff
@@ -35,7 +35,7 @@ default IndividualSignature findSignaturesFor(User user,  String claName) {
 		return results.isEmpty() ? null : results.get(0);
 	}
 
-	@Query("select s from IndividualSignature s where (s.cla.name = :#{#claName} or s.cla.name in (select distinct c.supersedingCla.name from ContributorLicenseAgreement c where c.name = :#{#claName})) and (s.githubLogin = :#{#u.githubLogin} or s.email in (:#{#u.emails}))")
+	@Query("select s from IndividualSignature s where (s.cla.name = :#{#claName} or s.cla.name in (select distinct c.supersedingCla.name from ContributorLicenseAgreement c where c.name = :#{#claName})) and (s.githubLogin = :#{#u.githubLogin} or s.email in (:#{#u.emails.empty ? '' : #u.emails}))")
```

Comment:
Don't expect Github sends wrong data, rather someone performs manual database actions/imports/exports data so data comes into the database through a different channel than we'd expect. 

That's a query which can be broken easily by bringing data into an unexpected state. Other changes to the databases likely produce somewhere an Exception but not a false positive.